tests/integration/test_wazuh_db/test_wazuh_db.py

```diff
@@ -77,3 +115,40 @@ def test_wazuh_db_create_agent(configure_sockets_environment, connect_to_sockets
                            "stage": "Syscheck - Agent does not exits yet"}]}
     test_wazuh_db_messages(configure_sockets_environment, connect_to_sockets_module, test['test_case'])
     assert os.path.exists(os.path.join(WAZUH_PATH, 'queue', 'db', "999.db"))
+
+
+def test_wazuh_db_chunks(configure_sockets_environment, connect_to_sockets_module):
+    """Check that commands by chunks work properly when agents amount exceed the response maximum size"""  
+    AGENTS_CANT = 14000
+    AGENTS_OFFSET = 20
+    AGENTS_MAX = AGENTS_OFFSET+AGENTS_CANT
+    
+    #Pre insert agents
+    insert_agents(AGENTS_OFFSET, AGENTS_CANT)
```

Comment:
If this function is not part of the test (it seems to be a prerequisite), it should be a fixture called by this particular test.

Reply:
Done